api/rest/v1/bulk_observations_series.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: default
-title: <bulk-tag>BULK</bulk-tag> Series of Observations
+title:  Series of Observations
```

Comment:
wonder if the extra space before "Series" matters?

Reply:
Don't think it matters, but cleaned up the extra space!